dev-packages/dynamic-require/index.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
```

Comment:
Could you explain why we need a *.d.ts and a *.js file instead of just writing typescript?

Reply:
It felt simplier to setup for such a trivial package. Tell me if you'd rather want compiled Typescript.